container_meals/tests/common.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2022 Coop IT Easy SCRL fs
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import SavepointCase
+
+
+class TestCommon(SavepointCase):
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
```

Comment:
why is the setup done at the class lever instead of at the object level?

edit: ok, [it’s a `SavepointCase` thing](https://www.odoo.com/documentation/14.0/developer/reference/addons/testing.html#odoo.tests.common.SavepointCase).